corporate/finance/990ez.rst

```diff
@@ -1,6 +1,18 @@
-990
----
+:og:description:
 
-Foundation Group
+Form 990
+--------
 
-Georgia department of revenue - via LetterStream
+.. vale Google.LyHyphens = NO
+.. vale Google.Ordinal = NO
+.. vale Google.Passive = NO
+.. vale Google.Units = NO
+.. vale Google.We = NO
+.. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+
+The Internal Revenue Service requires exempt organizations to file `Form 990 <https://www.irs.gov/forms-pubs/about-form-990>`_ annually, 5 months after the accounting period ends. RoboJackets operates on a July-June fiscal year, so we must file by November 15th each year. RoboJackets typically files Form 990-EZ.
+
+Form 990 is complex, and we've engaged Foundation Group to prepare and file it for most years since incorporating. Contact Foundation Group on July 1st each year to begin the process.
```

Comment:
```suggestion
Form 990 is complex, so we've engaged Foundation Group to prepare and file it for most years since incorporating. Contact Foundation Group on July 1st each year to begin the process.
```